invenio_orcid/utils.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Implement helper functions."""
+
+from invenio_oauthclient.models import RemoteAccount, UserIdentity
+
+
+def get_authors_credentials(orcid):
+    """Return ORCID access token for a specific author (if available)."""
+    raw_user = UserIdentity.query.filter_by(
+        id=orcid, method='orcid').first()
```

Comment:
@jmartinm what I meant is that it could be called 'orcid' in INSPIRE and 'Orcid' on Zenodo. So we should not hardcode it here :)